Backend/UnitTesting/UserServiceUT.cs

```diff
@@ -381,7 +381,7 @@ public void Is_User_Manager_Over_Independent()
 
             using (_db = tu.CreateDataBaseContext())
             {
-                Assert.IsFalse(us.IsManagerOver(_db, unassociatedUser, subject));
+                //Assert.IsFalse(us.IsManagerOver(_db, unassociatedUser, subject));
```

Comment:
Please remove these tests completely before PR. These shouldn't exist in the system anymore (applies to tests you commented below as well).

Reply:
Do you want the class deleted or just those specific tests?

Reply:
This has been fixed on the bug branch